modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -182,7 +183,9 @@ public List<ServiceRegistration<?>> deploy(
 		try {
 			ObjectDefinitionResourcePermissionUtil.populateResourceActions(
 				_objectActionLocalService, objectDefinition,
-				_portletLocalService, _resourceActions);
+				(ObjectDefinitionPersistence)
```

Comment:
Because this method is called also [here](https://github.com/liferay/liferay-portal/blob/71e32c86bc859261ff147cdf09e292a35d4af8ed/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java#L163-L166)
and in this class, is not possible call **objectDefinitionLocalService** because this will cause a [cyclic reference](https://github.com/liferay/liferay-portal/blob/5672c26c87d670b9372a97e33598c7bb7c419267/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java#L2401-L2402).